examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -359,6 +359,7 @@
 #include <IRtimer.h>
 #include <IRutils.h>
 #include <IRac.h>
+#include <RCSwitch.h> // RF record/decoding
```

Comment:
please, keep the new line

Reply:
Sorry, I did't get it. Add a new empty line?

Reply:
actually it is fine (I don't know why I wrote this, I guess I was in a bad reviewer day heh)